protostar/commands/test/test_command.py

```diff
@@ -1,4 +1,6 @@
-from logging import getLogger
+# pylint: disable=too-many-arguments
```

Comment:
Could we disable this lint globally and remove all these comments? Ofc topic for separate PR.

Reply:
https://github.com/software-mansion/protostar/issues/599